### DISCUSSION TITLE
Table组件如何实现鼠标hover到一行显示tooltip

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ant-design/ant-design#42532] [Q&A] [answered]
Table组件如何实现鼠标hover到一行显示tooltip，用原生js加事件可行，但是太麻烦了，有没有更便捷的方式

**Comment by linxianxi** [comment, marked as answer]
<img width="239" alt="image" src="https://github.com/ant-design/ant-design/assets/47104575/16becdf5-303d-4351-8c04-1322b033aafb">
